### PR TITLE
POW-338

### PR DESCRIPTION
**Pull request by OptimusOpus**
https://dltx.atlassian.net/jira/software/projects/POW/boards/48/backlog?assignee=61a957cd744c4d006930718e&selectedIssue=POW-338

json from https://bitcoinbrisbane-my.sharepoint.com/:x:/g/personal/tom_dodson_dltx_io/ETuart1iuaFAmR-WXZ-2lZABVri_ums0gpqbwPbFGUgo-g?e=cfl5QH

### CODE REVIEW

**Review by brettvniekerk — APPROVED** [review]
Reviewed on the excel sheet so LGTM

File: scripts/deploy/contracts/InvestorVesting.ts [hunk 1]
```diff
@@ -6,39 +6,151 @@ import hre, { ethers } from "hardhat";
...
   {
-    beneficiary: "0x042BC2D085c0584Bd56D62C170C4679e1ee9FC45",
-    vestingAmount: ethers.BigNumber.from("11150000"),
-    duration: 9000,
+    beneficiary: "0x470D968F0d27075F3Db0f28d2C2a5a6EEbaD9E65",
+    vestingAmount: ethers.BigNumber.from("750000000"),
+    duration: 63072000,
+    claimedAmount: 0,
+    lastClaimedTime: 0,
+    initialAmount: ethers.BigNumber.from("300000000"),
```
Comment (Mitchell139) [inline scripts/deploy/contracts/InvestorVesting.ts]: These should all be big numbers right?
```suggestion
    initialAmount: ethers.BigNumber.from("300000000000000000000000000"),
```

**Review by xiangzhe0812 — APPROVED** [review]

File: scripts/deploy/contracts/InvestorVesting.ts [hunk 2]
```diff
@@ -6,39 +6,151 @@ import hre, { ethers } from "hardhat";
...
+    claimedAmount: 0,
+    lastClaimedTime: 0,
+    initialAmount: ethers.utils.parseEther("0"),
+    initialClaimed: true,
+    claimStartTime: startDate + 0
+  },
+  {
+    beneficiary: "0xD8C128747183AF6EbdA860A2ea9B77b3933c25b4",
+    vestingAmount: ethers.utils.parseEther("500000000"),
+    duration: 157680000,
```
Comment (Mitchell139) [inline scripts/deploy/contracts/InvestorVesting.ts:78]: Should include cliff
```suggestion
    duration: 189216000,
```

File: scripts/deploy/contracts/InvestorVesting.ts [hunk 3]
```diff
@@ -6,39 +6,151 @@ import hre, { ethers } from "hardhat";
...
+    claimedAmount: 0,
+    lastClaimedTime: 0,
+    initialAmount: ethers.utils.parseEther("300000000"),
+    initialClaimed: false,
+    claimStartTime: startDate + 0
+  },
+  {
+    beneficiary: "0xb5408c2977daa3E371279b72bD598000db84d12a",
+    vestingAmount: ethers.utils.parseEther("1115000000"),
+    duration: 63072000,
```
Comment (Mitchell139) [inline scripts/deploy/contracts/InvestorVesting.ts:28]: Should include cliff
```suggestion
    duration: 94176000,
```

File: scripts/deploy/contracts/InvestorVesting.ts [hunk 4]
```diff
@@ -6,39 +6,151 @@ import hre, { ethers } from "hardhat";
...
-    claimStartTime: startDate
+    claimStartTime: startDate + 0
   },
   {
-    beneficiary: "0xEBE3efcfD01e27c1Ea8E48f5d55550C6325237af",
-    vestingAmount: ethers.BigNumber.from("30000000"),
-    duration: 18000,
+    beneficiary: "0xF2aE68ee7F461Ed24f96D089b6b48579EDbd7E4a",
+    vestingAmount: ethers.utils.parseEther("60000000"),
+    duration: 31536000,
```
Comment (Mitchell139) [inline scripts/deploy/contracts/InvestorVesting.ts:138]: Should include cliff 
```suggestion
    duration: 63072000,
```

**Review by Mitchell139 — CHANGES_REQUESTED** [review]

**Review by Mitchell139 — APPROVED** [review]
Apparently I'm remembering how the cliff time works incorrectly

**Pull request merged**